src/components/ShowProductsSection/ShowProductsSection.js

```diff
@@ -10,13 +11,13 @@ const ShowProductsSection = () => {
   };
 
   return (
-    <section className="showContainer">
-      <h2>Our Products</h2>
-      <p>
-        Lorem ipsum dolor sit amet, consectetur adipisicing elitsed do eiusmo
-        tempor incididunt ut labore
-      </p>
-      <div className="showContainer__btnContainer">
+    <section className="show-container">
+      <SectionHeader
+        title="Our Products"
+        description="Lorem ipsum dolor sit amet, consectetur adipisicing elitsed do eiusmo
+        tempor incididunt ut labore"
```

Comment:
Long strings you can put to separate variable and then pass this variable to component as a prop